editor/src/templates/editor-canvas.tsx

```diff
@@ -871,10 +876,9 @@ export class EditorCanvas extends React.Component<EditorCanvasProps> {
 
     const canvasIsLive = isLiveMode(this.props.editor.mode)
 
-    const canvasControls = React.createElement(NewCanvasControls, {
-      windowToCanvasPosition: this.getPosition,
-      cursor,
-    })
+    const canvasControls = (
```

Comment:
so I am only reading this code and haven't tried it yet, but doesn't this mean this will become _another_ div that is mounted a bit higher in the tree?

Reply:
This was always there. Balint has just updated this to use JSX syntax rather than `React.createElement`